upb/pb/encoder.h

```diff
@@ -64,6 +64,8 @@ const upb_handlers *upb_pb_encoder_newhandlers(const upb_msgdef *m,
 upb_sink *upb_pb_encoder_input(upb_pb_encoder *p);
 upb_pb_encoder* upb_pb_encoder_create(upb_env* e, const upb_handlers* h,
                                       upb_bytessink* output);
+void upb_pb_encoder_encode_unknown(upb_pb_encoder* p, const char* buf,
```

Comment:
I think this declaration and definition are unused?

Reply:
Renamed to encode_unknown. This needs to be exposed for php extension to write custom handler to encode bytes into unknown fields.